ui/src/message_popup/author/questions_page.jsx

```diff
@@ -29,12 +30,18 @@ export default React.createClass({
 
   getInitialState(){
     return ({
-      questions: allQuestions,
+      allDatabaseQuestions: [],
+      currentQuestions: [],
+      archivedQuestions: [],
```

Comment:
Choose a single representation for this state, rather than having multiple.  It's up to you whatever shape of data works best (eg., a single list, a map with two lists, two lists) but avoid storing two different projections of the same data in component `state`.

I think it probably makes the most conceptual sense to treat this as a single entity (either a single list or a map with two lists), since it will be loaded as one thing from the server.